utils/aws/s3.ts

```diff
@@ -0,0 +1,41 @@
+import aws from "aws-sdk";
+import fs from "fs";
+import path from "path";
+
+const S3 = new aws.S3();
+
+/* 
+
+Uploads a file to the AWS S3 bucket corresponding to AWS_BUCKET_NAME in .env.
+    * filePath: the path of the file to upload
+    * returns: upload response if successful, error object otherwise
+
+Sample usage:
+    myFile = "~/documents/my_file.txt"
+    res = uploadToS3(myFile)
+
+*/
+
+const uploadToS3 = (filePath: string): void => {
+    aws.config.update({
+        accessKeyId: process.env.AWS_ACCESS_KEY,
+        secretAccessKey: process.env.AWS_SECRET_KEY,
+        region: process.env.AWS_REGION,
+        signatureVersion: "v4",
+    });
+
+    const fileStream = fs.createReadStream(filePath);
+    const uploadParams = {
+        Bucket: process.env.AWS_BUCKET_NAME,
+        Key: path.basename(filePath),
+        Body: fileStream,
+    };
+    S3.upload(uploadParams, function (err, data) {
+        if (err) {
+            return err;
+        }
+        return data;
+    });
+};
+
+export { S3, uploadToS3 };
```

Comment:
why does `S3` need to be exported?

Reply:
My initial comment was saying we can initialize and export s3 object from `aws/index.js`. We can also just initialize in here outside of the function instead too. Either one works.

Reply:
I didn't mean export it out of this file